test/integration/consul-container/test/README.md

```diff
@@ -0,0 +1,67 @@
+# Upgrade Integration Tests
+
+The upgrade integration test is responsible for ensuring that users are able to upgrade between consul versions without experiencing difficulties. The upgrade tests does the following:
+- Ensure that customers are able to upgrade to a new Consul version and new and existing features work as expected. 
+- Test for backwards compatibility. i.e support for upgrading directly from Consul `1.13` to `1.15`, without upgrading to `1.14`
```

Comment:
Please explicitly list we upgrade test n-2 and n-1 versions and support skip-version upgrade.